### PR TITLE
321 demo review 0

### LINKED ISSUE #321: Demo review pre-r1
# Issue description
Review the existing demos - due 27th (sorry).

--------------

@RemiDelaunay 
classical_mr_prostate_nonrigid
paired_mrus_brain

@ZheMin-1992 
grouped_mask_prostate_longitudinal
unpaired_ct_abdomen

@agrimwood 
grouped_mr_heart
paired_mrus_prostate

@QianyeYang:
classical_ct_headneck_affine
unpaired_ct_lung

@NMontanaBrown 
paired_ct_lung
unpaired_mr_brain

@mathpluscode 
unpaired_us_prostate_cv

--------------

## Things to check:
- If it runs;
- In readme: text, links, clarity of the language;
- In the text in Introduction to DeepReg: https://github.com/DeepRegNet/DeepReg/blob/master/docs/source/demo/introduction.rst

## Things to suggest:
- Visualisation and results: how to point to the generated results; whether additional ones needed;
- What makes all demos have a consistent style;
- ANYTHING else!

## How to review
Preferably, create a branch and a [WIP] PR, such that the change is clear.
If in doubt, leave a comment below.

## Type of Issue

Please delete options that are not relevant.

- [ ] Bug report
- [ ] New feature request
- [ ] Documentation update

### PR DESCRIPTION
**Pull request by YipengHu**
# Description

- Demo doc review
- Demo readme review
- Make a consistent style for the demo readme and doc
- Add pretrained model to unpaired-us 

Fixes #321 

## Type of change

- [ ] Documentation Update (fix or improvement on the documentation)
- [ ] Demo

### CODE REVIEW

**Review by ebonmati — APPROVED** [review]

File: demos/classical_ct_headneck_affine/README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 This is a special demo that uses the DeepReg package for classical affine image
 registration, which iteratively solves an optimisation problem. Gradient descent is used
 to minimise the image dissimilarity function of a given pair of moving anf fixed images.
 
 ## Author
 
-DeepReg Development Team (raise an issue:
-https://github.com/DeepRegNet/DeepReg/issues/new, or mailto the author:
-yipeng.hu@ucl.ac.uk)
+DeepReg Development Team
```
Comment (mathpluscode) [inline demos/classical_ct_headneck_affine/README.md:14]: maybe it's still better to put your name? @YipengHu 

later with more demos, the proportion of your name will decrease lol

Comment (YipengHu) [reply demos/classical_ct_headneck_affine/README.md:14]: I will put my name then.... :D

Comment (YipengHu) [reply demos/classical_ct_headneck_affine/README.md:14]: i would encourage to raise ticket instead of going to individuals.

Comment (mathpluscode) [reply demos/classical_ct_headneck_affine/README.md:14]: yes, but we just gave name without giving email.

if you want, you can merge the author & contact section lol

Comment (YipengHu) [reply demos/classical_ct_headneck_affine/README.md:14]: maybe, after this PR

**Pull request merged**